pkg/subctl/benchmark/latency.go

```diff
@@ -118,4 +119,14 @@ func runLatencyTest(f *framework.Framework, testParams benchmarkTestParams) {
 
 	framework.By(fmt.Sprintf("Waiting for the client pod %q to exit, returning what client sent", nettestClientPod.Pod.Name))
 	nettestClientPod.AwaitFinishVerbose(Verbose)
+	nettestClientPod.CheckSuccessfulFinish()
+	latencyHeaders := strings.Split(nettestClientPod.TerminationMessage, "\n")[1]
+	headers := strings.Split(latencyHeaders, ",")
+	latencyValues := strings.Split(nettestClientPod.TerminationMessage, "\n")[2]
+	values := strings.Split(latencyValues, ",")
+	for i, v := range headers {
```

Comment:
Prints vertical table

```
Minimum Latency Microseconds:	50
Mean Latency Microseconds:	79.09
Maximum Latency Microseconds:	10893
Stddev Latency Microseconds:	88.03
Transaction Rate Tran/s:	12599.844

Minimum Latency Microseconds:	91
Mean Latency Microseconds:	142.28
Maximum Latency Microseconds:	9801
Stddev Latency Microseconds:	124.74
Transaction Rate Tran/s:	7006.188
```

Reply:
That must have been `intra-cluster == false`  where it runs 2 latency tests. So that's all it prints sans verbose? I think we need some context prior to each run by always printing

`framework.By("Performing latency tests from Gateway pod to Gateway pod")`

and

`framework.By("Performing latency tests from Non-Gateway pod to Non-Gateway pod")`

Reply:
These lines are printed only when verbose=true as per current implementation
framework.By("Performing latency tests from Gateway pod to Gateway pod")
		runLatencyTest(f, latencyTestParams)